packages/components-props-analyzer/package.json

```diff
@@ -15,7 +15,7 @@
         "compile": "node ../../scripts/build",
         "gen:props": "ts-node --project ./bin/tsconfig.json ./bin/index.ts",
         "prettify": "prettier --write 'src/components/*.ts'",
-        "start": "node ../../scripts/start"
+        "start": "nodemon --watch 'src/ComponentsList.ts' --exec 'pnpm build'"
```

Comment:
did you picked nodemon to help with the hot reload when changing props in the code? 👀

Reply:
Without nodemon, it wouldn't rerun the `gen:props` script, so adding a new component in the ComponentLists file wouldn't generate props for this new component.

Reply:
<img src="https://media2.giphy.com/media/0FHJ363Octui8Emuul/giphy.gif"/>